src/controllers/user/createUser.js

```diff
@@ -0,0 +1,12 @@
+import { Request, Response } from "express";
```

Comment:
I think you can remove it, please verify if the both imports are interfaces or class from express. Look at handle function signature below, its declare the same name but don't use these imports. Try comment this line and check if keep working. 
```suggestion
//import { Request, Response } from "express";
```

PS: I don't have sure about it.